moveit_ros/visualization/robot_state_rviz_plugin/src/robot_state_display.cpp

```diff
@@ -169,11 +169,6 @@ void RobotStateDisplay::changedEnableCollisionVisible()
   robot_->setCollisionVisible(enable_collision_visible_->getBool());
 }
 
-static bool operator!=(const std_msgs::msg::ColorRGBA& a, const std_msgs::msg::ColorRGBA& b)
```

Comment:
Why are we removing this?

Reply:
https://github.com/AcutronicRobotics/moveit2/pull/127/files/c0f4e01e76871b9f158be5600ff8562005465fe4#r302404706